### PR TITLE
version bump

### PR DESCRIPTION
**Pull request by julien-tierny**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by julien-tierny** [comment]
@conda-forge-admin, please rerender

**Comment by julien-tierny** [comment]
@lhofmann I'm trying this on my own this time ;)

**Comment by julien-tierny** [comment]
@conda-forge-admin, please rerender

**Comment by julien-tierny** [comment]
@lhofmann I find the azure error messages a bit cryptic :(
do you have any clue of what's going on?

**Comment by julien-tierny** [comment]
@lhofmann I find the azure error messages a bit cryptic :(
do you have any clue of what's going on?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -144,18 +144,17 @@ about:
     triangulations, in 2D, 3D, or more. It provides a substantial collection
     of generic, efficient and robust implementations of key algorithms in
     topological data analysis. It includes:
-    · For scalar data: critical points, integral lines, persistence diagrams,
-      persistence curves, merge trees, contour trees, Reeb graphs, Morse-Smale
-      complexes, topological simplification, topology-aware compression,
-      harmonic design;
-    · For bivariate scalar data: fibers, fiber surfaces, continuous
+    - For scalar data: critical points, integral lines, persistence diagrams,
+persistence curves, merge trees, contour trees, Reeb graphs, Morse-Smale complexes, topological simplification, topology-aware compression, harmonic design;
```
Comment (lhofmann) [inline recipe/meta.yaml:148]: All lines after `  description: |` need to be indented with the same number of spaces, so you probably need to add four spaces in front of this line.

Comment (julien-tierny) [reply recipe/meta.yaml]: thanks a lot, I'm on it :)

**Comment by lhofmann** [comment]
FTR, the windows builds were failing with error `undefined symbol: __kmpc_dispatch_deinit`, which apparently was a bug in llvm 18 (https://github.com/llvm/llvm-project/issues/98046).

Conda-forge llvm was just updated to v19 a few hours ago (https://github.com/conda-forge/openmp-feedstock/pull/144), so I re-started the CI so it uses the latest llvm version.

**Comment by julien-tierny** [comment]
awesome, thanks a lot @lhofmann

**Comment by julien-tierny** [comment]
@conda-forge-admin, please rerender

**Pull request merged**